src/order/cover.lean

```diff
@@ -236,6 +239,9 @@ instance covby.is_irrefl : is_irrefl α (⋖) := ⟨λ a ha, ha.ne rfl⟩
 lemma covby.Ioo_eq (h : a ⋖ b) : Ioo a b = ∅ :=
 h.wcovby.Ioo_eq
 
+lemma covby_iff_Ioo : a ⋖ b ↔ a < b ∧ Ioo a b = ∅ :=
```

Comment:
Maybe worth a more explicit name (that matches `Ioo_eq` above)
```suggestion
lemma covby_iff_lt_and_Ioo_eq : a ⋖ b ↔ a < b ∧ Ioo a b = ∅ :=
```

Reply:
I think
```suggestion
lemma covby_iff_Ioo_eq : a ⋖ b ↔ a < b ∧ Ioo a b = ∅ :=
```
is explicit enough.